### PR TITLE
Documentation - add links to JS samples 

### LINKED ISSUE #470: Documentation for using Graph SDK with MSAL.js
# Feature Request

## Is your feature request related to a problem? Please describe

We have implemented a set of samples (see below) demonstrating how to use MSAL.js as a custom authentication provider for this library. Would it be possible to document these or mention them in the wiki?

- [Vanilla JS SPA using MSAL Browser](https://github.com/Azure-Samples/ms-identity-javascript-tutorial/tree/main/2-Authorization-I/1-call-graph)
- [Angular SPA using MSAL Angular](https://github.com/Azure-Samples/ms-identity-javascript-angular-tutorial/tree/main/2-Authorization-I/1-call-graph)
- [React SPA using MSAL React](https://github.com/Azure-Samples/ms-identity-javascript-react-tutorial/tree/main/2-Authorization-I/1-call-graph)
 
[AB#9843](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/9843)

### PR DESCRIPTION
**Pull request by nikithauc**
Fixes #470

### CODE REVIEW

**Review by MIchaelMainer — APPROVED** [review]

**Review by MIchaelMainer — CHANGES_REQUESTED** [review]
The sendUsersMail image and link is a bit off. Should probably be in a section about typings.

Also, there are duplicate links to samples and tutorials at the bottom of the document. Perhaps that is a good thing, it is nice to find the links in the TOC.

**Comment by nikithauc** [comment]
> The sendUsersMail image and link is a bit off. Should probably be in a section about typings.
> 
> Also, there are duplicate links to samples and tutorials at the bottom of the document. Perhaps that is a good thing, it is nice to find the links in the TOC.

@MIchaelMainer can you explain what should be the correction be for the sendUsersMail image and link? 

The duplicate links in the TOC and the samples sections are intentional.

File: README.md [hunk 1]
```diff
@@ -29,6 +29,20 @@ The Microsoft Graph JavaScript client library is a lightweight wrapper around th
         -   [Page Iteration](docs/tasks/PageIterator.md)
         -   [Getting Raw Response](docs/GettingRawResponse.md)
         -   [Creating an instance of TokenCredentialAuthenticationProvider](docs/TokenCredentialAuthenticationProvider.md)
+    - [Samples and tutorials](#samples-and-tutorials)
+        - Step-by-step training exercises on creating a basic application using the Microsoft Graph JavaScript SDK:
+            - [Build Angular single-page apps with Microsoft Graph](https://docs.microsoft.com/en-us/graph/tutorials/angular)
```
Comment (baywet) [inline README.md:34]: nit: if you remove en-us from the links, people will get redirected to their language if it's available

**Comment by MIchaelMainer** [comment]
@nikithauc The sendUsersMail image and link occur in the same section as that TOC/Navigation section. There should a section and description that describe what the image is and why the customer should follow it.

Perhaps something like:

## TypeScript

This client library is supported with typings that represent the objects in Microsoft Graph. We provide typing support for the [v1.0](https://github.com/microsoftgraph/msgraph-typescript-typings) and [beta](https://github.com/microsoftgraph/msgraph-beta-typescript-typings) models.

[TypeScript demo](https://github.com/microsoftgraph/msgraph-typescript-typings)

**Comment by nikithauc** [comment]
> @nikithauc The sendUsersMail image and link occur in the same section as that TOC/Navigation section. There should a section and description that describe what the image is and why the customer should follow it.
> 
> Perhaps something like:
> 
> ## TypeScript
> This client library is supported with typings that represent the objects in Microsoft Graph. We provide typing support for the [v1.0](https://github.com/microsoftgraph/msgraph-typescript-typings) and [beta](https://github.com/microsoftgraph/msgraph-beta-typescript-typings) models.
> 
> [TypeScript demo](https://github.com/microsoftgraph/msgraph-typescript-typings)
Done!

**Review by MIchaelMainer — APPROVED** [review]

**Pull request merged**